core/cli/command.js

```diff
@@ -0,0 +1,122 @@
+const cli = require('@tryghost/pretty-cli');
```

Comment:
I think this dependency is hoisted from gscan's dependencies and we need to add it to package.json here too